pkg/config/validation/validation.go

```diff
@@ -1939,8 +1939,9 @@ func ValidateControlPlaneAuthPolicy(policy meshconfig.AuthenticationPolicy) erro
 	return fmt.Errorf("unrecognized control plane auth policy %q", policy)
 }
 
-func validateWorkloadSelector(selector *type_beta.WorkloadSelector) error {
+func validateWorkloadSelector(selector *type_beta.WorkloadSelector) Validation {
```

Comment:
There is actually 2 types of WorkloadSelector, the other is validateAlphaWorkloadSelector which looks like already has this warning. So LGTM